### ISSUE TITLE
Runoff perturbation

### ISSUE THREAD

**Issue by adele157** [adele157/IAF-meltwater#1]
Make some plots showing the runoff perturbation that was applied. e.g.:
* Masks of the 3 sector domains.
* Timeseries of runoff in the 3 sectors.
* Distribution of control/perturbation runoff with longitude, ala Figure 3a from [Moorman et al. 2020](https://journals.ametsoc.org/view/journals/clim/33/15/jcliD190846.xml). We'd have to pick a time slice/average for this one (e.g. 2018? or average over whole period?).

**Comment by adele157** [comment]
The runoff was applied based on the decadal averages of grounding line flux from [Rignot et al. 2019](https://www.pnas.org/doi/10.1073/pnas.1812883116). 

We didn't add any additional meltwater to account for ice shelf thinning, because the control runoff (from Depoorter et al. 2013) includes ice shelf thinning estimates from Pritchard et al. 2009. The Pritchard et al. 2009 ice shelf thinning rates are pretty dodgy, because they are only based on data from 2003-2007 and have been shown to have quite different trends from the longer term trends (Paolo et al. 2015). But in the regions where we added meltwater, the shorter term ice shelf thinning rates generally at least have the same sign as the longer term trends.

**Comment by adele157** [comment]
Meltwater was added to 3 different regions (Peninsula (which includes WAP and Larsen), Amundsen/Bellingshausen, and Wilkes) with different rates of increase for each region, based on curves fitted to the decadal Rignot estimates. For each of these 3 regions, the control runoff was scaled according to the curves below. We decided not to apply the meltwater increase on a smaller scale (i.e. using individual ice shelf trends) because 1. that would have been harder to implement, 2. there is large along-shelf connectivity, so perhaps it wouldn't matter too much anyway, and 3. There are large uncertainties on individual ice shelves.

These are the 3 regions where extra meltwater was applied (white regions have no extra meltwater beyond the control value):
<img width="600" alt="Screen Shot 2022-08-18 at 11 37 58 am" src="https://user-images.githubusercontent.com/8506963/185353746-d0c8440a-9e83-45ed-b4d5-45b70fb8446f.png">

Time series applied to the Peninsula region (as a percent of the control runoff in this region and in Gt/y). Red dots are the Rignot values.
<img width="700" alt="Screen Shot 2022-08-18 at 11 52 01 am" src="https://user-images.githubusercontent.com/8506963/185353795-aa61b77b-761f-4a02-98a5-f421124e5187.png">

Time series applied to the Amundsen/Bellingshausen region:
<img width="700" alt="Screen Shot 2022-08-18 at 11 52 09 am" src="https://user-images.githubusercontent.com/8506963/185353844-99e0cb4d-4f29-456f-bb47-dc03cad89b3e.png">

Constant additional runoff in the Wilkes sector (i.e. an extra 6% runoff is applied every year compared to the control):
<img width="700" alt="Screen Shot 2022-08-18 at 11 52 20 am" src="https://user-images.githubusercontent.com/8506963/185353906-c54f9b12-24ca-4fbb-a117-7a16e3855f38.png">
